font/src/ft/mod.rs

```diff
@@ -618,8 +622,9 @@ impl FreeTypeRasterizer {
         }
 
         for fallback_font in &fallback_list.list {
+            let font_hash = fallback_font.hash;
             let font_pattern = &fallback_font.pattern;
```

Comment:
Why do we extract these two variables here? Is there any specific reason like a linebreak or something that we prevent by putting things here? Or would this work just as well by inlining these two variables?

Reply:
borrow checker.